csrc/ops/arith.cpp

```diff
@@ -2617,4 +2634,67 @@ TensorView* prefixSum(TensorView* tv, int64_t dim) {
       /*init=*/tv->fusion()->zeroVal(tv->dtype()));
 }
 
+BlockQuantizationResults blockQuantize(TensorView* input) {
+  auto reshaped_input = reshape(input, [](auto& x) { x.split(-1, 16); });
```

Comment:
Please do not just silently place an op like this. Please give a reason.

Reply:
Please take a look at this.